exercise/static/exercise/results_staff.js

```diff
@@ -0,0 +1,840 @@
+    exerciseSelectionChange();
+});
+
+$("#avg-p-checkbox").change(function() {
+    _avgPTrue = this.checked ? true : false;
+    exerciseSelectionChange();
+});
+
+$("#max-p-checkbox").change(function() {
+    _maxPTrue = this.checked ? true : false;
+    exerciseSelectionChange();
+});
+
+
+// Set up the tooltip for checkboxes, look docs below for tooltip
+// https://getbootstrap.com/docs/3.3/javascript/
+$('[data-toggle="tooltip"]').tooltip({
+    "trigger": "hover",
+});
+
+$('[data-toggle="tooltip"]').on('click', function() {
+    $(this).tooltip('hide');
+})
+
+
+// Event listener for tag filters
+$('.filter-users button').on('click', function(event) {
+    event.preventDefault();
+    let icon = $(this).find('.glyphicon');
+    if (icon.hasClass('glyphicon-unchecked')) {
+        icon.removeClass('glyphicon-unchecked').addClass('glyphicon-check');
+    } else {
+        icon.removeClass('glyphicon-check').addClass('glyphicon-unchecked');
+    }
+    exerciseSelectionChange();
+});
+
+
+/**
+ * Creates the html for indicator data row. This is called for each row that has 
+ * checkbox checked for its indicator data.
+ * @param  {string} tooltipTitle The text that is shown when hovering over heading.
+ * @param  {string} headingTitle The heading text.
+ * @return {string} Returns the html that creates the data indicator rows in table.
+ */
+function createIndicatorRow(tooltipTitle, headingTitle, dataValues) {
+// Data indicator headings have title and tooltip info that is shown when hovering mouse on the heading title
+    let indicatorHeadingHtml = "";
+    if (dataValues && dataValues.length > 0) {
+        indicatorHeadingHtml +=
+            '<tr class="no-filtering"><td style="border-right: none !important; border-bottom: none !important;"' +
+            'class="indicator-heading stick-on-scroll" data-toggle="tooltip" ' +
+            'title="' + tooltipTitle + '">' + headingTitle +
+            '</td><td style="border-left: none !important; border-bottom: none !important;"' +
+            'class="indicator-heading stick-on-scroll" data-toggle="tooltip" ' +
+            'title="' + tooltipTitle + '"></td>';
+    } else {
+        indicatorHeadingHtml +=
+            '<tr class="no-filtering"><td style="border-right: none !important;" colspan="2"' +
+            'class="indicator-heading stick-on-scroll" data-toggle="tooltip" ' +
+            'title="' + tooltipTitle + '">' + headingTitle +
+            '</td><td style="border-left: none !important; border-top: none !important;"' +
+            'class="indicator-heading stick-on-scroll" data-toggle="tooltip" ' +
+            'title="' + tooltipTitle + '"></td>';        
+    }
+
+    let normalValuesHtml = '';
+    let pctValuesHtml = 
+        '<tr class="no-filtering tableexport-ignore">'
+    let sumValue = 0;
+    dataValues.forEach(function(value) {
+        sumValue += value[0];
+    });
+
+// Empty data cells for total and tags in data indicators
```

Comment:
Why these are not indented?

Reply:
Some style guide recommended putting all comments starting from left margin. Not sure what I should follow.

Reply:
sounds pretty interesting style guide. this seemed pretty nice https://github.com/airbnb/javascript  with exceptions on those features that are not supported by ie 11.

Reply:
https://www.crockford.com/code.html

This was the style guide I looked at for comments. I assume the rest are same as in airbnb guide aside from comments, which I changed.